debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: coding-shell-extensions
+Source: <url://example.com>
+
+Files: *
+Copyright: <years> <put author's name and email here>
```

Comment:
Hm, right, might be tedious to fill this if this means adding copyrights for the specific extensions. Is some debian program complaining if you just remove those things?